source/virtualBuffers/__init__.py

```diff
@@ -109,14 +109,12 @@ def obj(self):
 
 	@property
 	def label(self):
-		if self.itemType == "landmark":
-			attrs = self.textInfo._getControlFieldAttribs(self.vbufFieldIdentifier[0], self.vbufFieldIdentifier[1])
-			name = attrs.get("name", "")
-			if name:
-				name += " "
-			return name + aria.landmarkRoles[attrs["landmark"]]
-		else:
-			return super(VirtualBufferQuickNavItem,self).label
+		value = super(VirtualBufferQuickNavItem,self).label
+		if self.itemType is "heading":
+			# Explicitly handle headings here as well, to avoid requesting the Control field attribs

```

Comment:
This comment says "as well". As well as what? Unless I'm missing something, "as well" should be removed.